packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json

```
@@ -5,6 +5,10 @@
   "telephony_line_tab_consumption": "Consommation",
   "telephony_line_tab_call_management": "Gestion des appels",
   "telephony_line_tab_music_management": "Gestion des musiques",
+  "telephony_line_tab_softphone": "Softphone",
+  "telephony_line_tab_beta_softphone_description": "Paramétrez ici votre option Softphone. A noter que pendant notre phase Béta, si vous choisissez d'avtiver le Softphone sur votre ligne, nous ne pourrez plus la provisionner ailleurs que sur nos applications de Softphone, « Softcall » (et donc pas sur un terminal de téléphonie fixe ou sur un autre logiciel de Softphonie)",
+  "telephony_line_tab_beta_softphone_active_description": "Si l'interrupteur est actif, alors cette ligne est utilisable sur l'ensemble de vos applications « Softcall »: mobile et ordinateur",
+  "telephony_line_tab_beta_softphone_inactive_description": "Si l'interrupteur est gris, alors cette ligne est utilisable uniquement sur un terminal de téléphonie fixe (filaire ou DECT); ou sur un softphone de votre choix",
```

```suggestion
  "telephony_line_tab_beta_softphone_inactive_description": "Si l'interrupteur est inactif, alors cette ligne est utilisable uniquement sur un terminal de téléphonie fixe (filaire ou DECT) ou sur un softphone de votre choix.",
```
To maintain logic in activation